ykshim_client/src/lib.rs

```diff
@@ -0,0 +1,245 @@
+//! Client to the ykshim crate in the internal workspace.
```

Comment:
This file is a good place to start a review. It shows the interface to the internal workspace.